apps/public/src/styles/landing/_footer.scss

```
@@ -0,0 +1,71 @@
+  .landing-footer-link {
+    color: var(--mat-sys-on-surface);
+    font-size: .875rem;
+    line-height: 1.5;
+    text-decoration-color: var(--mat-sys-outline);
+    text-decoration-thickness: 1px;
+    text-underline-offset: .2em;
+  }
+
+  .landing-footer-link:hover {
+    color: var(--mat-sys-primary);
+    text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)